SimPEG/electromagnetics/static/induced_polarization/simulation_2d.py

```diff
@@ -22,40 +24,38 @@ class BaseIPSimulation2D(BaseDCSimulation2D):
 
     eta, etaMap, etaDeriv = props.Invertible("Electrical Chargeability (V/V)")
 
+    data_type = properties.StringChoice(
+        "IP data type",
+        default='volt',
+        choices=['volt', 'apparent_chargeability'],
```

Comment:
what do you think of using the dictionary for string choice here (to allow the user to input common names and coerce them) 
```
choices={
    "volt": ["voltage"],
    "apparent chargeability": ["apparent_chargeability", "apparent-chargeability", "apparentchargeability"]
}
```